osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -41,6 +41,7 @@ const (
 	debianImage   = "projects/debian-cloud/global/images/family/debian-9"
 	centosImage   = "projects/centos-cloud/global/images/family/centos-7"
 	rhelImage     = "projects/rhel-cloud/global/images/family/rhel-7"
+	windowsImage  = "projects/windows-cloud/global/images/family/windows-2016"
```

Comment:
What's the plan for extending this to all the versions we want to support?

Reply:
I would suggest making all these []string and then ranging over them for creating tests

Reply:
this will get complicated with your current setup, you may want to use some sort of config struct like we have for inventory tests

Reply:
+1. But let's do that as a separate PR.

Reply:
i actually looked into it. with the current setup, looks like, it is going to get complicated. will send out a separate PR